ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -580,6 +456,15 @@ open class SegmentedControl: UIControl {
         button.isSelected = isSelected
     }
 
+    @objc private func themeDidChange(_ notification: Notification) {
+        if let window = self.window,
```

Comment:
Can you change this to a guard let to conform to the swift guid on early return?

https://github.com/microsoft/swift-guide/blob/gh-pages/EarlyReturn.md